### PR TITLE
feat(bedrock): add anchor focus styles

### PR DESCRIPTION
**Pull request by Francois-Esquire**
## Description

Updating `css-bedrock` with new focus treatment for `<a>` tags.

## Detail

- Modifies the postcss configuration to use native `:focus-visible`
- Updates bedrock CSS to include focus styles for anchor with a `2px` `outline` (instead of `box-shadow`)
- Transition and color weights have parity with `react-components`

focus-visible

## Checklist

* [ ] :ok_hand: style updates are Garden Designer approved (add the
  designer as a reviewer)
* [ ] :globe_with_meridians: ~component demo is up-to-date (`yarn start`)~
* [ ] :white_check_mark: ~all component states are represented
  (`.is-hovered`, `.is-focused`, etc.)~
* [x] :metal: renders as expected sans Bedrock (`?bedrock=false`)
* [x] :memo: tested in Chrome, Firefox, Safari, Edge, and IE11

### CODE REVIEW

File: packages/bedrock/src/_base.css [hunk 1]
```diff
@@ -48,6 +48,17 @@ a:focus {
 
 a:focus { outline: none; }
 
+a:focus-visible {
+  outline: calc(var(--zd-spacing-xxs) - 2px) solid var(--zd-color-blue-600);
+  outline-offset: calc(var(--zd-spacing-xxs) - 3px);
```
[the remark this answers is not in the document]
Comment (jzempel) [reply packages/bedrock/src/_base.css:53]: Also compare https://github.com/zendeskgarden/css-components/blob/04d0ba3b981a4f9c3fc3d70744c4b55874c4920d/packages/buttons/src/_color.css#L8. In other words, a change in spacing should not impact the focus styling (clearly theming over here isn't as rich as `react-components` – but that's fine and not in scope atm).

[the remark this answers is not in the document]
Comment (Francois-Esquire) [reply packages/bedrock/src/_base.css:53]: Agreed, this is a bit forced. I will use literal values and update this!

**Review by jzempel — CHANGES_REQUESTED** [review]
The anchor should get color updates for `:hover` and `:active` here so that it has parity with `react-components`.

File: packages/bedrock/src/_base.css [hunk 2]
```diff
@@ -34,19 +34,38 @@ html {
...
-a:focus {
+a:hover {
   text-decoration: underline;
   color: var(--zd-anchor-color);
 }
 
-a:focus { outline: none; }
+a:hover,
+a:active {
+  color: var(--zd-color-blue-800);
```
Comment (jzempel) [inline packages/bedrock/src/_base.css:57]: This does not match `react-components`. https://github.com/zendeskgarden/react-components/blob/e1a2f869c466c4917d4668bc24e2acf42773f178/packages/buttons/src/styled/StyledButton.ts#L67-L70 Hover is blue-700; active is blue-800.

**Review by jzempel — CHANGES_REQUESTED** [review]

File: packages/bedrock/src/_base.css (hunk 2 above)
Comment (Francois-Esquire) [reply packages/bedrock/src/_base.css:57]: That should be corrected now!

**Review by jzempel — APPROVED** [review]

**Review by lucijanblagonic — APPROVED** [review]
With JZ's comments addressed, LGTM!

**Pull request merged**